.buildkite/pipelines/quality-gates/pipeline.kibana-tests.yaml

```diff
@@ -0,0 +1,5 @@
+steps:
+  - label: ":pipeline::grey_question::seedling: Trigger Kibana Tests for ${ENVIRONMENT}"
+    command: "make run-environment-tests"
```

Comment:
Just to make sure I understand the relationship:

1. First `run-environment-tests` inside this `Makefile` is executed: https://github.com/elastic/ci-agent-images/blob/main/container-images/platform-qx/quality-gate-seedling/Makefile#L2-L4
2. From there `run-buildkite-quality-gate.sh` is executed: https://github.com/elastic/ci-agent-images/blob/main/container-images/platform-qx/quality-gate-seedling/run-buildkite-quality-gate.sh
3. Which finally loads the environment specific yaml file, for example  `.buildkite/pipelines/quality-gates/pipeline.test-qa.yaml`

Is that correct?

Reply:
Yes or in other words. 
The ` pipeline.kibana-tests.yaml` is the entry and seeds the actual tests per environment by branching to the environment specific pipeline definitions in:
```
pipeline.qa-tests.yaml                      // will be executed after QA version was bumped
pipeline.staging-tests.yaml              // etc...

```

Reply:
yes.  Thats the current convention, everyone needs to have a `pipeline.tests-<env>.yaml` file....I need to add a prod one...